extensions/wikia/AdEngine/js/AdEngineRunner.js

```diff
@@ -19,9 +19,11 @@ define('ext.wikia.adEngine.adEngineRunner', [
 	 * @param {function} runAdEngine
 	 */
 	function delayRun(runAdEngine) {
-		var enabledBidders = [],
-			biddersQueue = [],
-			startedByBidders = false;
+		var biddersQueue = [],
+			enabledBidderNames = [],
```

Comment:
@fraszczakszymon Is it what you wanted? https://github.com/Wikia/app/commit/41b7d974d45eb57b510499e6ede5238b7efb52c8
:smiley_cat: